### ISSUE TITLE
Passing props

### ISSUE THREAD

**Issue by mhaagens** [jtmthf/react-router-match-async#2]
How would one go about passing in props? I.E. I used to be able to;
`render={(props) => <Home {...props} store={store} />}` in RR4.

**Comment by jtmthf** [comment]
Good question. This is a use case I haven't thought of yet. As the component to be rendered needs to be dynamically loaded, it should be passed as a parameter to the render function. I'm thinking of an api that would look like this:

``` javascript
<MatchAsync
  pattern="/"
  getComponent={loadHome}
  render={(Home, props) => <Home {...props} store={store} />}
/>
```

This should be a pretty easy addition that I can add to the next release.

**Comment by mhaagens** [comment]
Yeah, that looks good! I'll be eagerly awaiting the next release then 👍
